R/get_publication.R

```diff
@@ -0,0 +1,61 @@
+#' Get publications
```

Comment:
Would it be possible to let the users not care about the pages, and instead we loop over and query as many times as needed to give them a full data set?

Would expect the output to be just the $results item but as a data frame

Reply:
Done and done